packages/cloud_firestore_odm/cloud_firestore_odm/example/integration_test/cloud_firestore_odm_e2e_test.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import 'collection_reference_test.dart' as collection_reference_test;
+import 'document_reference_test.dart' as document_reference_test;
+import 'path_test.dart' as path_test;
+import 'query_reference_test.dart' as query_reference_test;
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
```

Comment:
afaik this isn't required, because the flutter tool creates a temporary file where the call to `IntegrationTestWidgetsFlutterBinding.ensureInitialized()` is automatically inserted. See https://github.com/flutter/flutter/issues/92254 for more details :)

Reply:
Thanks for the note, @bartekpacia. However, it seems that without `IntegrationTestWidgetsFlutterBinding.ensureInitialzed()` the tests doesn't on web. Maybe it's related with https://github.com/flutter/flutter/issues/66264

See:
* https://github.com/firebase/flutterfire/actions/runs/3213776146/jobs/5253698990
* https://github.com/firebase/flutterfire/actions/runs/3213776149/jobs/5253699165
* https://github.com/firebase/flutterfire/actions/runs/3213776150/jobs/5253699260